### ISSUE TITLE
Add support for Hist Gradient Boosting from scikit-learn

### ISSUE THREAD

**Issue by StrikerRUS** [BayesWitnesses/m2cgen#381]
https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingRegressor.html
https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.HistGradientBoostingClassifier.html

**Comment by StrikerRUS** [comment]
Added into feature-request hub #491.
